roles/os_hardening/tasks/minimize_access.yml

```diff
@@ -80,3 +80,28 @@
     fstype: proc
     opts: '{{ proc_mnt_options }}'
     state: mounted
+
+- name: Harden mount options for tmp
```

Comment:
I'd like these tasks reversed, so we first remount it with hardened options and then write everything to `/etc/fstab`. This way we do not persist options, that will lead to failing mount of `/tmp`